crates/common/src/runtime_client.rs

```diff
@@ -113,6 +117,7 @@ impl RuntimeClient {
         timeout: Duration,
         compute_units_per_second: u64,
         jwt: Option<String>,
+        headers: Vec<String>,
```

Comment:
So, just to not get blocked on this let's just ignore the clippy lint for now—however a good followup would be making a `RuntimeClientBuilder` type that can build a `RuntimeClient` so we avoid this (and make the `RuntimeClient` members public).

Reply:
Fixed with builder